definitions/infrastructure_as_a_service.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Infrastructure as a Service (IaaS)
+status: Completed
+category: Concept
+---
+
+## Infrastructure as a Service
+
+### What it is
+
+Infrastructure as a service i.e. IaaS is a [cloud computing](https://github.com/cncf/glossary/blob/main/definitions/cloud_computing.md) service model that offers either [physical](https://github.com/cncf/glossary/blob/main/definitions/bare_metal_machine.md) or [virtualized](https://github.com/cncf/glossary/blob/main/definitions/virtualization.md) compute, storage, and network resources on-demand over the internet to consumers on a pay-as-you-go cost model.
+In the IaaS model, cloud providers own and operate the hardware and software, it is available in Public Cloud, Private Cloud, and Hybrid Cloud deployment options. 
+
+### Problem it addresses
+
+Organizations often struggle with effective computing resource utilization in a traditional on-premise setup. The data center has to be built for perceived peak demand. When the workload declines, computing resources would be idle. When the workload spikes beyond the perceived demand, there is a shortage of computing resources to process the workload. This lack of scalability leads to increased cost and a lack of effective resource utilization.
+
+### How it helps
+
+Organizations no longer need to purchase and manage their equipment and the physical space required to host the equipment, it is both costly and labor-intensive in a traditional on-premise setup.  With infrastructure being made available on demand by cloud providers, it reduces the need for high upfront CAPEX, while giving the flexibility to scale up or down IT resources based on demand. It also saves time and cost of provisioning and scaling new development and test environments, which helps developers to experiment and innovate.
```

Comment:
Made significant edits. @kumar-deni please let me know if you're comfortable with the changes. If so I'm ready to merge.

Reply:
@JasonMorgan - looks good..please go ahead..